### PR TITLE
🌐 Add Spanish translation for `docs/tutorial/response-status-code.md`

### PR DESCRIPTION
**Pull request by carlosm27**
Hi folks, I hope all of you are well.

According to the Spanish Translation feature: https://github.com/tiangolo/fastapi/issues/1218
This pull request includes the Spanish translation for tutorial/response-status-code.md

Thanks!

### CODE REVIEW

**Comment by carlosm27** [comment]
I don't know why tests failed. I saw the test's details and they say the test fails in the file `test/test_tutorial/test_async_tests/test_main.py`. Can someone explain to me why this happened? I think I didn't change other files but the one I was working on.

File: docs/es/docs/tutorial/response-status-code.md [hunk 1]
```diff
@@ -0,0 +1,105 @@
...
+      y la respuesta no debe tener cuerpo.
+
+- `300` en adelante son para "Redirección". Respuestas con estos códigos de estado pueden o no tener body, excepto para `304`, "No Modificado",
+  el cual no debe tener body.
+
+- `400` en adelante son para respuestas para `Error del Cliente`. Estos son el segundo tipo de codigós que más usarás.
+     - Un ejemplo es `404`, para respuesta de "No encontrado".
+     - Para errores genéricos desde el cliente, puedes usar `400`.
+
+- `500` en adelante son para errores del servidor. Tú nunca los usas directamente. Cuando algo sale mal en alguna parte del código de tu aplicación,
```
Comment (erudin) [inline docs/es/docs/tutorial/response-status-code.md:66]:
```suggestion
- `500` en adelante son para errores del servidor. Tú casi nunca los usas directamente. Cuando algo sale mal en alguna parte del código de tu aplicación,
```

File: docs/es/docs/tutorial/response-status-code.md [hunk 2]
```diff
@@ -0,0 +1,105 @@
...
+
+Pero tú no tienes que memorizar lo que significa cada uno de estos códigos.
+
+Puedes usar las convenientes variables desde `fastapi.status`.
+
+```Python hl_lines="6"
+{!../../../docs_src/response_status_code/tutorial002.py!}
+```
+
+Son tan convenientes, mantienen el mismo número, pero puedes usar el autocompletado del editor para encontrarlas:
```
Comment (erudin) [inline docs/es/docs/tutorial/response-status-code.md:91]:
```suggestion
Son solo una conveniencia, mantienen el mismo número, pero puedes usar el autocompletado del editor para encontrarlas:
```

File: docs/es/docs/tutorial/response-status-code.md [hunk 3]
```diff
@@ -0,0 +1,105 @@
...
+
+<img src="../../../en/docs/img/tutorial/response-status-code/image02.png">
+
+!!! note "Detalles técnicos"
+    Podrías usar también `from starlette import status`.
+
+
+```
+**FastAPI** provee el mismo `starlette.status` como  `fastapi.status`  conveniente para tí, el desarrollador. Pero viene directamente de Starlette.
```
Comment (erudin) [inline docs/es/docs/tutorial/response-status-code.md:100]:
```suggestion
**FastAPI** provee el mismo `starlette.status` como  `fastapi.status` solo como una conveniencia para tí, el desarrollador. Pero viene directamente desde Starlette.
```

File: docs/es/docs/tutorial/response-status-code.md [hunk 4]
```diff
@@ -0,0 +1,105 @@
...
+```
+
+# Cambiando el por defecto
+
+Después, en la [Guía Avanzada de Usuario](https://github.com/carlosm27/fastapi/blob/master/docs/en/docs/advanced/response-change-status-code.md), verás cómo retornar diferentes códigos de estado que los de por defecto declaras aquí.
```
Comment (erudin) [inline docs/es/docs/tutorial/response-status-code.md:105]:
```suggestion
Después, en la [Guía Avanzada de Usuario](https://github.com/carlosm27/fastapi/blob/master/docs/en/docs/advanced/response-change-status-code.md), verás cómo retornar un código de estado diferente al predeterminado que declaras aquí.
```

**Review by erudin — COMMENTED** [review]
Looks good! Thanks! Just a few small comments regarding missing accents and some rephrasing for clarity.

File: docs/es/docs/tutorial/response-status-code.md [hunk 5]
```diff
@@ -0,0 +1,105 @@
...
+- Documentado como en el OpenAPI "schema" (además, en la interfaz de usuario):
+
+<img src="../../../en/docs/img/tutorial/response-status-code/image01.png">
+
+
+!!! note
+    Algunos códigos de respuesta (ve la siguiente sección) indica que la respuesta no tiene body.
+
+```
+FastAPI sabe esto, y producirá documentatión OpenAPI diciendo que no hay body en la respuesta.
```
[the remark this answers is not in the document]
Comment (chesstrian) [reply docs/es/docs/tutorial/response-status-code.md:38]:
```suggestion
FastAPI sabe esto, y producirá documentación OpenAPI diciendo que no hay cuerpo en la respuesta.
```

File: docs/es/docs/tutorial/response-status-code.md [hunk 6]
```diff
@@ -0,0 +1,105 @@
...
+
+!!! note
+    Toma en cuenta que `status_code` es un parámetro del método del "decorador" (`get`, `post`, etc)-
+No de la función de tu *path operation*, como todos los parámetros y cuerpo.
+
+
+El parámetro `status_code` recibe un número con el código del estado HTTP.
+
+!!! info
+   `status_code` puede alternamente recibir un `IntEnum`, como <a href="https://docs.python.org/3/library/http.html#http.HTTPStatus" class="external-link" target="_blank">`http.HTTPStatus`</a> de Python.
```
Comment (chesstrian) [inline docs/es/docs/tutorial/response-status-code.md:24]:
```suggestion
   `status_code` puede alternativamente recibir un `IntEnum`, como <a href="https://docs.python.org/3/library/http.html#http.HTTPStatus" class="external-link" target="_blank">`http.HTTPStatus`</a> de Python.
```

File: docs/es/docs/tutorial/response-status-code.md [hunk 7]
```diff
@@ -0,0 +1,105 @@
...
+    Si ya sabes lo que son los códigos de estado HTTP, salta a la siguiente sección.
+
+En HTTP, tu envías un código de estado de tres dígitos como parte de la respuesta.
+
+Estos códigos de estado tienen un nombre asociado para reconocerlos, pero lo más importante es el número.
+
+En resumen:
+
+ - `100` en adelante son para "Información". Rara vez los usas directamente. Las respuestas con estos códigos de estado no tienen body.
+ - `200` en adelante son para respuestas "Exitosas". Esta son las que se usan con mayor frecuencia.
```
Comment (chesstrian) [inline docs/es/docs/tutorial/response-status-code.md:53]: Here we are talking about "códigos de estado":
```suggestion
 - `200` en adelante son para respuestas "Exitosas". Estos son las que se usan con mayor frecuencia.
```

File: docs/es/docs/tutorial/response-status-code.md [hunk 8]
```diff
@@ -0,0 +1,105 @@
...
+- `400` en adelante son para respuestas para `Error del Cliente`. Estos son el segundo tipo de codigós que más usarás.
+     - Un ejemplo es `404`, para respuesta de "No encontrado".
+     - Para errores genéricos desde el cliente, puedes usar `400`.
+
+- `500` en adelante son para errores del servidor. Tú nunca los usas directamente. Cuando algo sale mal en alguna parte del código de tu aplicación,
+  o en el servidor, automáticamente retornará uno de estos códigos.
+
+!!! tip
+    Para saber más acerca de estos códigos de estado y cuál código se usa para cada caso, revisa la
+<a href="https://developer.mozilla.org/en-US/docs/Web/HTTP/Status" class="external-link" target="_blank"><abbr title="Mozilla Developer Network"> documentación acerca de los códigos de estados MDN</abbr> .
```
Comment (chesstrian) [inline docs/es/docs/tutorial/response-status-code.md:71]: It says "códigos de estado" in the previous line:
```suggestion
<a href="https://developer.mozilla.org/en-US/docs/Web/HTTP/Status" class="external-link" target="_blank"><abbr title="Mozilla Developer Network"> documentación acerca de los códigos de estado MDN</abbr> .
```

File: docs/es/docs/tutorial/response-status-code.md (hunk 4 above)
Comment (chesstrian) [reply docs/es/docs/tutorial/response-status-code.md:105]: Isn't there any problem with the link to the forked repo?

**Review by chesstrian — CHANGES_REQUESTED** [review]

**Review by chesstrian — CHANGES_REQUESTED** [review]
The entry for the new translated page is missing from `docs/es/mkdocs.yml`.

**Comment by chesstrian** [comment]
@carlosm27 is it intended not to make changes over `docs/es/mkdocs.yml`?

**Comment by carlosm27** [comment]
Hey @chesstrian. Honestly, I don't know. I think we make changes when a topic is added.

**Comment by carlosm27** [comment]
Hey @chesstrian  Thank you for sharing this. I didn't know about updating that file.

**Review by chesstrian — APPROVED** [review]

**Comment by tiangolo** [comment]
This was covered in another PR, so I'll close this one now, thanks for the effort! :cake: :coffee:

**Pull request closed without merging**